test/e2e/basic_provisioning_test.go

```diff
@@ -78,13 +78,6 @@ var _ = Describe("Provisioning", func() {
 		err = clusterProxy.GetClient().Create(ctx, &bmh)
 		Expect(err).NotTo(HaveOccurred())
 
-		By("Waiting for the BMH to be in registering state")
-		WaitForBmhInProvisioningState(ctx, WaitForBmhInProvisioningStateInput{
-			Client: clusterProxy.GetClient(),
-			Bmh:    bmh,
-			State:  metal3api.StateRegistering,
-		}, e2eConfig.GetIntervals(specName, "wait-registering")...)
-
```

Comment:
FYI (other reviewers): This was removed because the `registering` state passes so quickly that it is sometimes missed in the fixture tests. I think it is not important to check that we go through this state since it is the next state we really care about (`inspecting` or `available`).

Reply:
Yep, I remember having this concern when looking at the e2e patch.